content/docs/ai/_index.md

```diff
@@ -0,0 +1,78 @@
+
+### Prompt enrichment
+
+Prompts are basic building blocks for guiding LLMs to produce relevant and accurate responses. By effectively managing both system prompts, which set initial guidelines, and user prompts, which provide specific context, you can significantly enhance the quality and coherence of the model's outputs. AI Gateway allows you to pre-configure and refactor system and user prompts, extract common AI provider settings so that you can reuse them across requests, dynamically append or prepend prompts to where you need them, and overwrite default settings on a per-route level.
+
+### Prompt guards
+
+Prompt guards are mechanisms that ensure that prompt-based interactions with a language model are secure, appropriate, and aligned with the intended use. These mechanisms help to filter, block, monitor, and control LLM inputs and outputs to filter offensive content, prevent misuse, and ensure ethical and responsible AI usage. With AI Gateway, you can set up prompt guards to block unwanted requests to the LLM provider and mask sensitive data.
+
+### Chat streaming
+
+AI Gateway supports chat streaming, which allows the LLM to stream out tokens as they are generated. The way that chat streaming is determined varies by AI provider.
+
+* OpenAI and most AI providers: Most providers send the `is-streaming` boolean as part of the request to determine whether or not a request should receive a streamed response. 
+* Google Gemini and Vertex AI: In contrast, the Gemini and Vertex AI providers change the path to determine streaming, such as the `streamGenerateContent` segment of the path in the Vertex AI streaming endpoint `https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-flash-latest:streamGenerateContent?key=<key>`. To prevent the path you defined in your HTTPRoute from being overwritten by this streaming path, you instead indicate chat streaming for Gemini and Vertex AI by setting `spec.options.ai.routeType=CHAT_STREAMING` in your RouteOptions resource.
+
+## Supported LLM providers {#supported-llm-providers}
+
+The examples throughout the AI Gateway docs use OpenAI as the LLM provider, but you can use other providers supported by AI Gateway.
+
+{{< reuse "docs/snippets/product-name-caps.md" >}} supports the following AI providers:
+
+* [Anthropic](https://docs.anthropic.com/en/release-notes/api)
+* [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/)
+* [Gemini](https://ai.google.dev/gemini-api/docs)
+* [OpenAI](https://platform.openai.com/docs/overview). You can also use `openai` support for LLM providers that use the OpenAI API, such as [DeepSeek](https://api-docs.deepseek.com/) and [Mistral](https://docs.mistral.ai/getting-started/quickstart/).
+* [Vertex AI](https://cloud.google.com/vertex-ai/docs)
+
+Note the following differences in how AI Gateway features function for each provider.
```

Comment:
i think remove if there are no differences in how theyre implemented in kgateway